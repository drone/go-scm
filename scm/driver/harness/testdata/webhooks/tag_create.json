{
  "trigger": "tag_created",
  "repo": {
    "id": 16,
    "path": "kmpySmUISimoRrJL6NL73w/harness-core",
    "identifier": "harness-core",
    "default_branch": "develop",
    "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "uid": "harness-core"
  },
  "principal": {
    "id": 2,
    "uid": "lv0euRhKRCyiXWzS7pOg6g",
    "display_name": "Admin",
    "email": "admin@harness.io",
    "type": "user",
    "created": 1701091219051,
    "updated": 1701091219051
  },
  "ref": {
    "name": "refs/tags/asd",
    "repo": {
      "id": 16,
      "path": "kmpySmUISimoRrJL6NL73w/harness-core",
      "identifier": "harness-core",
      "default_branch": "develop",
      "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
      "uid": "harness-core"
    }
  },
  "sha": "3e4da2d65c3631c3e84b52fabe714c978aff540b",
  "head_commit": {
    "sha": "0f1835abe08473e07863540712d8389984b72dad",
    "message": "",
    "author": {
      "identity": {
        "name": "admin",
        "email": "admin@harness.io"
      },
      "when": "2024-03-01T07:54:35-08:00"
    },
    "committer": {
      "identity": {
        "name": "GitHub",
        "email": "noreply@github.com"
      },
      "when": "2024-03-01T07:54:35-08:00"
    },
    "added": [],
    "removed": [],
    "modified": []
  },
  "commit": {
    "sha": "0f1835abe08473e07863540712d8389984b72dad",
    "message": "",
    "author": {
      "identity": {
        "name": "Jenny James",
        "email": "jenny.james@harness.io"
      },
      "when": "2024-03-01T07:54:35-08:00"
    },
    "committer": {
      "identity": {
        "name": "GitHub",
        "email": "noreply@github.com"
      },
      "when": "2024-03-01T07:54:35-08:00"
    },
    "added": [],
    "removed": [],
    "modified": []
  },
  "old_sha": "0000000000000000000000000000000000000000",
  "forced": false
}