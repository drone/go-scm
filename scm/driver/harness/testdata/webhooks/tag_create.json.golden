{
    "Ref": {
       "Name": "refs/tags/asd",
       "Sha": "3e4da2d65c3631c3e84b52fabe714c978aff540b"
    },
    "Action": "created",
    "Repo": {
        "ID": "16",
        "Namespace": "",
        "Name": "harness-core",
        "Perm": null,
        "Branch": "develop",
        "Private": false,
        "Clone": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
        "CloneSSH": "",
        "Link": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
    "Sender": {
        "ID": "lv0euRhKRCyiXWzS7pOg6g",
        "Login": "lv0euRhKRCyiXWzS7pOg6g",
        "Name": "Admin",
        "Email": "admin@harness.io",
        "Avatar": "",
        "Created": "2023-11-27T05:20:19.051-08:00",
        "Updated": "2023-11-27T05:20:19.051-08:00"
    }
}